common/beerocks/bwl/dwpal/mon_wlan_hal_dwpal.cpp

```diff
@@ -72,6 +94,36 @@ static void calc_curr_traffic(const uint64_t val, uint64_t &total, uint32_t &cur
     total = val;
 }
 
+static bool dwpal_get_freq(const std::vector<unsigned int> &channel_pool, unsigned int curr_channel,
```

Comment:
Please add doxygen to this function. SInce it is static bool, it should be in the cpp file.

Reply:
But it is in the cpp file :)

Reply:
I meant that the doxygen comment should be in the cpp file

Reply:
added